server/controllers/email-report/weeklySummaryReport.handlebars

```diff
@@ -0,0 +1,118 @@
+{{> head title="PATIENT_RECORDS.FINANCIAL_ACTIVITY.REPORT_TITLE" }}
+
+<body>
+    <style type="text/css">
```

Comment:
Please do not define new styles in HTML.  If you want to change styles, do it in `bhima-bootstrap.less`.